pkg/templates/installation_context.go

```diff
@@ -16,24 +16,26 @@ type InstallationContext struct {
 	Logger log.Logger
 }
 
-func (ctx *InstallationContext) FuncMap() template.FuncMap {
-	return template.FuncMap{
-		"EntitlementValue": func(name string) string {
-			if ctx.Meta.Entitlements.Values == nil {
-				level.Debug(ctx.Logger).Log("event", "EntitlementValue.empty")
-				return ""
-			}
+func (ctx *InstallationContext) entitlementValue(name string) string {
```

Comment:
weird whitespace things happening here because I pulled this up to a standalone function instead of [inline](https://github.com/replicatedhq/ship/pull/884/files#diff-9c37f1a6dbeb3cee12d6bd0ce83693c7L21).